aws/basic-lambda/blueprint.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: xl/v2
+kind: Blueprint
+
+metadata:
+  name: AWS-Lambda-Basic
+  description: |
+    Creates a basic Lambda function and execution role
+  author: XebiaLabs
+  version: 2.0
+  instructions: Please read the generated file "xebialabs/USAGE-aws-basic-lambda.md" for further usage instructions.
+
+spec:
+  parameters:
+  # AWS specific variables
+  - name: UseAWSCredentialsFromSystem
+    type: Confirm
+    prompt: Do you want to use AWS credentials from ~/.aws/credentials file?
+    promptIf: !expr "awsCredentials('IsAvailable')"
+
+  - name: AWSAccessKey
+    type: SecretInput
+    prompt: What is the AWS Access Key ID?
+    promptIf: !expr "!UseAWSCredentialsFromSystem"
+    default: !expr "awsCredentials('AccessKeyID')"
+
+  - name: AWSAccessSecret
+    type: SecretInput
+    prompt: What is the AWS Secret Access Key?
+    promptIf: !expr "!UseAWSCredentialsFromSystem"
+    default: !expr "awsCredentials('SecretAccessKey')"
+
+  - name: AWSRegion #options are set manually because EKS service is not registered within AWS GO SDK
```

Comment:
comment is invalid. Also we can use `!expr "awsRegions('lambda')"` instead of hardcoding the options

Reply:
Yeah, I copied this from the other AWS blueprints. I didn't know about `awsRegions('lambda')`. Will try now

Reply:
Done. Although we lose out on the friendly names for the regions

Reply:
Unfriendly(but understandable) names are better than having to keep the list updated manually

Reply:
Agreed. I spoke to Erinç about maybe merging the friendly names in the `xl` binary itself as a possible feature.